mqt/qecc/cc_decoder/color_code.py

```diff
@@ -0,0 +1,61 @@
+"""A color code class."""
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import TYPE_CHECKING
+
+import numpy as np
+from ldpc import mod2
+
+if TYPE_CHECKING:  # pragma: no cover
+    import numpy.typing as npt
+
+
+class LatticeType(Enum):
```

Comment:
You should be able to make the string conversions a little easier with the following
```suggestion
class LatticeType(Enum, str):
```